### ISSUE TITLE
Plugins

### ISSUE THREAD

**Issue by alanef** [Hube2/contact-form-7-dynamic-select-extension#11]
Just quick note that I may be interested in your plugins.
I will have to look at the code / functionality in more detail, and right now I'm deep into a client project and supporting my existing plugins, so i'd appreciate a little time.

I'd also, like to discuss further what your reasoning is behind the transition arrangements you propose.

**Comment by Hube2** [comment]
Glad you are interested, and you can take your time. I completely understand client projects and time. It's one of the reasons why I'm having a hard time supporting something I do not use.

My concern with the transition has to do with emails I've gotten from WP Plugins about people giving plugins away to have them end up being used for hacking sites. I'd like to get a sense of who I am giving my work to, hope you understand, nothing personal. If you have ideas on this I'm willing to listen. Other than that, it seemed like it would be easier because it would happen over a period of time instead of all at once and allow me to time to deal with it.

**Comment by alanef** [comment]
Feel free to check me out  https://profiles.wordpress.org/fullworks#content-plugins
and https://fullworks.net

I'm probably as diagonally opposite to a hacker as you can get.  My plan would be to move to a freemium model if I could identify premium value adds, obviously thus supporting the free functionality and giving plugins a chance of a longer life / better support - I'm trying to move away from client driven projects to products. Feel free to contact me via email if you want alan at fullworks.net.

Will get back to when I get a chance to take a really good look.

**Comment by chaance** [comment]
Hi there. I am also interested in forking/helping here. Let me know @alanef and @Hube2 if you two settled on a plan last year. Let me know how I might be able to support.

**Comment by Hube2** [comment]
No, this never happened. If you are interested in the 3 plugins I'm still looking for someone to take them over. To be honest, I don't even know if they still work because I have not been keeping up with CF7.

**Comment by chaance** [comment]
@Hube2 Yes, I'm interested. This one works but the dynamic mail plugin does not, it seems (I just hacked them both into my own plugin so I could do what I needed, so I'm already familiar enough with the code). I'm happy to go ahead and fork them, and have you review if you want to work it that way to start.

**Comment by Hube2** [comment]
@chancestrickland sorry I did not get to you. If anyone wants to help maintain these just fork them. I will temporarily push any fixes to WP for now. If it works out and you want to continue to maintain them I will eventually transfer ownership of the WP SVN repo.

**Comment by PrimitivMedia** [comment]
@Hube2 Are you still looking for somebody to take over the plugins, I still see them under adopt me, I and my team are interested. Please advise and then we will fork as requested. Thank you!

**Comment by Hube2** [comment]
Sorry for the late reply, life has been a little crazy.

Yes, these plugins are still available for adoption.

I hope you understand that I'm a little leery about just giving them away before I'm sure that they will be maintained well.

**Comment by PrimitivMedia** [comment]
Hi John,

 

Life is hectic for everybody during these unforeseen times…no worries. What would you like I and my team to do to be able to adopt the plugin, we understand you simply cannot just give it to somebody.

 

Looking forward to hearing back from you!

 

Regards,

 

Nicolas Montigny

Founder & Website Developer | Primitiv Media

 <mailto:nicolas@primitiv.media> nicolas@primitiv.media |  <tel:226.344.9804> 226.344.9804 |  <https://primitiv.media/> primitiv.media

The following e-mail is intended for the recipients mentioned above, any access from anybody else is unauthorized. If you are not the intended recipient, any disclosure, copying, distribution or any action taken or omitted to be taken in regards to the content of this message is strictly prohibited and may be unlawful. If you have received this e-mail by mistake, please notify the sender immediately and delete the original message.

**Comment by Hube2** [comment]
I sent you an email on this but did not get a reply.

If you want to take over the 3 plugins.
1) Fork them here
2) Send me pull requests and I will push changes to WP SVN Repo
3) If that goes well for a while I will transfer ownership of the WP Repo

**Comment by PrimitivMedia** [comment]
Hi There,I had sent a reply stating that we probably won't be able to do much for around a month. That was a few weeks ago, in a few weeks we most likely will reach out and take a look at everything see what we can add. Sorry the fist email didn't make it to you.Stay safe.Nicolas-------- Original message --------From: "John A. Huebner II" <notifications@github.com>Date: Tue., May 5, 2020, 7:22 a.m.To: Hube2/contact-form-7-dynamic-select-extension <contact-form-7-dynamic-select-extension@noreply.github.com>Cc: Primitiv Media <nicolas@primitiv.media>, Comment <comment@noreply.github.com>Subject: Re: [Hube2/contact-form-7-dynamic-select-extension] Plugins (#11)
I sent you an email on this but did not get a reply.
If you want to take over the 3 plugins.
Fork them hereSend me pull requests and I will push changes to WP SVN RepoIf that goes well for a while I will transfer ownership of the WP Repo
